<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMedia</class>
 <widget class="QMainWindow" name="QMedia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMedia</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="start_buttton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forward_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>forward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QToolButton" name="volume_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>23</width>
      <height>25</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>静音</string>
    </property>
    <property name="text">
     <string>音量</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/images/volume.png</normaloff>:/images/volume.png</iconset>
    </property>
   </widget>
   <widget class="QListWidget" name="music_list">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="process_slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="volume_slider">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="start_player_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="process_label_2">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>270</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="mode_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>shuffle</string>
    </property>
   </widget>
   <widget class="QLabel" name="process_label_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
